### PR TITLE
New package: libetpan 1.6

### PR DESCRIPTION
**Pull request by skrzyp**
This is my third Void Linux template and first library template.

It contains libetpan, a library to access NNTP and IMAP4 servers, used by Claws Mail and other e-mail clients.

This is also a preparation for Claws Mail template.

### CODE REVIEW

File: srcpkgs/libetpan/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+pre_configure() {
+	./autogen.sh
+}
+
+libetpan-devel_package() {
+	depends="${pkgname}>=${version}_${revision}"
+		short_desc+=" - development files"
+		pkg_install() {
+			vmove usr/include
+				vmove "usr/lib/*.so"
```
Comment (dominikh) [inline srcpkgs/libetpan/template]: What's with the indentation here?

File: srcpkgs/libetpan/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+pkgname="libetpan"
+version="1.6"
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool automake autoconf"
+short_desc="Mail Framework library for C language"
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="BSD"
+homepage="http://www.etpan.org/libetpan.html"
+distfiles="https://github.com/dinhviethoa/libetpan/archive/1.6.tar.gz"
```
Comment (dominikh) [inline srcpkgs/libetpan/template]: use ${version} instead of 1.6 in the URL

**Comment by dominikh** [comment]
Squash all those commits

File: srcpkgs/libetpan/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+libetpan-devel_package() {
+	depends="${pkgname}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.la"
```
Comment (xtraeme) [inline srcpkgs/libetpan/template]: This line is wrong... libtool archives (*.la) are removed by default. I really doubt you have tested this.

You are also missing the soname change in common/shlibs.

**Comment by xtraeme** [comment]
If this pkg installs pkg-config files those should be part of the -devel pkg.

**Pull request closed without merging**